### ISSUE TITLE
Losing focus after insert text

### ISSUE THREAD

**Issue by lenhodder** [thundernest/quicktext#208]
I position the cursor at the point I want to insert text.  Then, I click the appropriate quicktext button, and the text is inserted.  In earlier versions, the focus remains in the body of the message, with the cursor right after the inserted text.  Now, however, the focus is elsewhere, and to add more text, I have to click the cursor after the newly inserted text in order to continue typing.

**Comment by SamuelPlentz** [comment]
I think this one is related to #197. The new Quicktext bar can be focused (the old one couldn't?!). So on selecting an option there the focus is on the Quicktext bar and gets restored correctly there.

As a workaround you can place a `[[CURSOR]]`-Tag into your template. Then the focus is set to the message body and not restored from the previous focused element.

I don't know what should be done.

- If previously the Quicktext bar was focused > focus the message body
- Focus the message body when there was any text to insert to the message body
- Add a Quicktext option to always focus the message body after using a template

**Comment by SamuelPlentz** [comment]
Please answer in the correct issue. You wrote in #197:
> adding the [[CURSOR]] tag isn't working for me. Maybe I have something wrong? See my signature HTML below.
> 
> <style></style>
> Cheers. <title></title>
> Len
> Cell: xxx-xxx-xxxx
> 
> 
> [[CURSOR]]

Couldn't reproduce the problem with this template. It works with for me:

```
<style></style>
Cheers. <title></title>
Len
Cell: xxx-xxx-xxxx


[[CURSOR]]
```

Perhaps something got scrambled/mixed up here. The use of `<style></style>` and `<title></title>` make no sense at all...

Can you post the correct template? Can you check if a text template works?

**Comment by SamuelPlentz** [comment]
Could you please confirm, that the attached version solves the problem?
[quicktext focus.zip](https://github.com/jobisoft/quicktext/files/5541008/quicktext.focus.zip)

**Comment by SamuelPlentz** [comment]
I decided to always focus the message body after using the Quicktext-toolbar. With the fix of #197 without `[[CURSOR]]` the toolbar is not selected anymore. (It is selected but one cannot reach it) So IMO it would be a good choice to always focus the message body in this case. (the pull request does that)

Other Quicktext insertion methods seem not to be affected.

**Comment by lenhodder** [comment]
That seems perfect!  Fixed issue 197 and 208 as far as I can tell.  Thanks for that!

**Comment by ghost** [comment]
Hi, how do I install the test version from the .zip file, please?

**Comment by jobisoft** [comment]
![grafik](https://user-images.githubusercontent.com/5830621/99505757-84c5af00-2981-11eb-89b9-f1f13f165df5.png)

and select the zip file

**Comment by ghost** [comment]
That's fixed it - many thanks.

**Comment by afabent** [comment]
This seems to work for me. Thank you!

**Comment by jobisoft** [comment]
Released as v3.5.
